lib/ramble/ramble/expander.py

```diff
@@ -799,8 +799,10 @@ def _eval_comparisons(self, node):
         """Handle a comparison node in the ast"""
 
         # Extract In nodes, and call their helper
-        if len(node.ops) == 1 and isinstance(node.ops[0], ast.In):
-            return self._eval_comp_in(node)
+        if len(node.ops) == 1 and (
+            isinstance(node.ops[0], ast.In) or isinstance(node.ops[0], ast.NotIn)
+        ):
+            return self._eval_comp_in_or_not(node, reverse=isinstance(node.ops[0], ast.NotIn))
```

Comment:
Can the reverse logic be encapsulated in the eval method?

Reply:
Oh right indeed, that'd be much simpler. Updated.